api/tests/app/email_branding/test_rest.py

```diff
@@ -13,11 +13,16 @@ def test_get_email_branding_options(admin_request, notify_db, notify_db_session)
         'email_branding.get_email_branding_options'
     )['email_branding']
 
-    assert len(email_branding) == 2
+    assert 6 == len(email_branding)
     assert {
         email_branding['id'] for email_branding in email_branding
     } == {
-        str(email_branding1.id), str(email_branding2.id)
```

Comment:
What's the reason there are now 6 brandings, not 2?

Reply:
With the db being recreated every time, there are migrations that adds rows into the `email_branding` table: https://github.com/govau/notify/blob/f786fd0cdf4fd9c7e69608705e1cca68be4fca71/api/migrations/versions/0161_email_branding.py#L33